<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Sidebar"/>
   </widget>
   <widget class="QMenu" name="menuItem">
    <property name="title">
     <string>&amp;Node</string>
    </property>
    <addaction name="actionNew_Node"/>
    <addaction name="actionNew_Excitory_Link"/>
    <addaction name="actionNew_Inhibitory_Link"/>
    <addaction name="separator"/>
    <addaction name="actionLabel"/>
    <addaction name="action_Delete"/>
    <addaction name="separator"/>
    <addaction name="actionActivate"/>
    <addaction name="actionDeactivate"/>
    <addaction name="actionToggleFrozen"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionStep"/>
    <addaction name="actionReset"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menuItem"/>
   <addaction name="menuSimulation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="action_Close"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="actionStep"/>
   <addaction name="actionReset"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="sidebarDockWidget">
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="sidebarContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolBox" name="sidebarToolbox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="sidebar_page_1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>79</width>
          <height>250</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Tools</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open Network...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close Network</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New Network...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save Network</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Sidebar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Sidebar</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionNew_Excitory_Link">
   <property name="text">
    <string>New &amp;Excitory Link</string>
   </property>
  </action>
  <action name="actionNew_Inhibitory_Link">
   <property name="text">
    <string>New &amp;Inhibitory Link</string>
   </property>
  </action>
  <action name="actionNew_Node">
   <property name="text">
    <string>New No&amp;de</string>
   </property>
  </action>
  <action name="actionLabel">
   <property name="text">
    <string>Label</string>
   </property>
  </action>
  <action name="actionActivate">
   <property name="text">
    <string>Activate</string>
   </property>
  </action>
  <action name="actionDeactivate">
   <property name="text">
    <string>Deactivate</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="text">
    <string>Step</string>
   </property>
  </action>
  <action name="actionToggleFrozen">
   <property name="text">
    <string>Toggle Frozen</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
